src/editors/EditorContainer.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable react/require-default-props */
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { getConfig } from '@edx/frontend-platform';
+
+import EditorPage from './EditorPage';
+
+interface Props {
+  /** Course ID or Library ID */
+  learningContextId: string;
+  /** Event handler for when user cancels out of the editor page */
+  onClose?: () => void;
+  /** Event handler for when user saves changes using an editor */
+  onSave?: () => (newData: Record<string, any>) => void;
```

Comment:
Nit: this happens after the save, not the actual save action.
```suggestion
  afterSave?: () => (newData: Record<string, any>) => void;
```

Reply:
Done: cdd2b63f70c4dfa47faa9396ecdd8df62567a446